man/man5/zfs-module-parameters.5

```diff
@@ -958,7 +958,7 @@ This limit is only enforced at module load time, and will be ignored if
 precedence over \fBzfs_dirty_data_max_max_percent\fR. See the section
 "ZFS TRANSACTION DELAY".
 .sp
-Default value: 25% of physical RAM.
+Default value: \fB25%\fR of physical RAM.
```

Comment:
I wonder if we should keep the % sign out of the bolding to prevent confusion as to whether the sysctls accept the character.